test/functional/feature_maxtipage.py

```diff
@@ -51,6 +52,10 @@ def run_test(self):
             self.log.info(f"Test IBD with maximum tip age of {hours} hours (-maxtipage={maxtipage}).")
             self.test_maxtipage(maxtipage)
 
+        max_long_val = 9223372036854775207
```

Comment:
nit: this is still wrong
```suggestion
        max_long_val = 9223372036854775807
```

Reply:
not that it matters for the test or otherwise, as the value can be anything that overflows, but it isn't the max long value